ExtractUserDataActivity/__tests__/handler.test.ts

```diff
@@ -125,36 +116,6 @@ describe("createExtractUserDataActivityHandler", () => {
     );
   });
 
-  it("should not export webhook notification data", async () => {
-    const notificationWebhookModelMock = ({
-      findNotificationsForMessage: jest.fn(() =>
-        createMockIterator([aRetrievedWebhookNotification])
-      )
-    } as any) as NotificationModel;
-
-    const handler = createExtractUserDataActivityHandler(
-      messageModelMock,
-      messageStatusModelMock,
-      notificationWebhookModelMock,
-      notificationStatusModelMock,
-      profileModelMock,
-      senderServiceModelMock,
-      blobServiceMock,
-      aUserDataContainerName
-    );
-    const input: ActivityInput = {
-      fiscalCode: aFiscalCode
-    };
-
-    await handler(contextMock, input);
-
-    // @ts-ignore as
-    const mockCall = createCompressedStream.mock.calls[0];
```

Comment:
this is wrong and we must rewrite it. if you mock the blobService and call cb() immediately the code following `createWriteStreamToBlockBlob` won't never be executed